src/app/core/ui/search/search.component.ts

```diff
@@ -97,7 +97,7 @@ export class SearchComponent implements OnInit {
 
     const searchTerms = searchString.split(" ");
 
-    const queryResults = await this.dbIndexingService.queryIndex(
+    const queryResults = await this.dbIndexingService.queryIndexRaw(
```

Comment:
```suggestion
    const queryResults = await this.dbIndexingService.queryIndexDocs<Entity>(
```
In a later step the results are transformed into entities, I did not find a reason why the raw results are needed

Reply:
I like the idea to hide `queryIndexRaw` (in fact I had been thinking the same) but the search component is currently transforming the results to either Child or School entities as they can be both. We could parse into any type based on the prefix but that would require a central register of all Entity types (probably something using Angular DI with `multi: true` would be great).

As the priority is finally getting the event attendance epic merged, I would argue to postpone all this a bit.

Reply:
I can agree to postponing it. Making the search component more flexible is not really required yet I gues